aws/cf_template.go

```diff
@@ -268,10 +274,17 @@ func generateTemplate(spec *stackSpec) (string, error) {
 	})
 
 	if spec.loadbalancerType == LoadBalancerTypeApplication && spec.wafWebAclId != "" {
-		template.AddResource("WAFAssociation", &cloudformation.WAFRegionalWebACLAssociation{
-			ResourceArn: cloudformation.Ref("LB").String(),
-			WebACLID:    cloudformation.String(spec.wafWebAclId),
-		})
+		if strings.HasPrefix(spec.wafWebAclId, "arn:aws:wafv2") {
+			template.AddResource("WAF2WAC1KPI3", &cloudformation.WAFv2WebACLAssociation{
```

Comment:
What does this name mean? `WAF2WAC1KPI3` can we use something more human readable? :)

Reply:
Good point

I got it from the visual template, but it can be renamed

Changed it to the same as from v1 `WAFAssociation`